app/jenkins-for-jira-ui/src/components/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PageHeader from '@atlaskit/page-header';
+import { ButtonGroup } from '@atlaskit/button';
+import Button from '@atlaskit/button/standard-button';
+import { headerContainer } from '../JenkinsServerList/JenkinsServerList.styles';
+import { TopPanel } from './TopPanel/TopPanel';
+import { mainPageContainer } from './MainPage.styles';
+
+const MainPage = (): JSX.Element => {
+	const pageHeaderActions = (
+		<ButtonGroup>
+			{/* TODO - add onClick event */}
```

Comment:
Only working on UI until it's all complete. Then I'll circle back and start adding functionality